content/learn/quick-start/getting-started/apps.md

```diff
@@ -8,13 +8,16 @@ weight = 2
 
 Bevy programs are referred to as [`App`]s. The simplest Bevy app looks like this:
 
-```rs,hide_lines=1
-# use bevy::prelude::*;
+```rs
+use bevy::prelude::*;
+
 fn main() {
     App::new().run();
 }
 ```
 
+The statement `use bevy::prelude::*` brings in the essential things from Bevy. However, since it is boilerplate it will be ommitted from all examples where it is irrelevant from here on out.
```

Comment:
```suggestion
The statement `use bevy::prelude::*` brings in the essential things from Bevy. However, since it is boilerplate it will be omitted from all examples where it is irrelevant from here on out.
```

A typo, but I might suggest something like

> The `use bevy::prelude::*` statement brings in the essential things from Bevy. For brevity, this guide may omit it in later steps.